app/views/components/_panel.html.erb

```diff
@@ -0,0 +1,13 @@
+<%
```

Comment:
What is this "panel" component going to be used for?
Is it something that could end up in govuk_publishing_components?
Can you add some tests for it?

Reply:
Tests added